merqube_client_lib/templates/equity_baskets/create_index.py

```diff
@@ -37,18 +37,18 @@
 def main(index_type: str, config_file_path: str, prod_run: bool) -> None:
     """main entrypoint"""
     assert os.path.exists(config_file_path), f"Config file path does not exist: {config_file_path}"
+    func: Callable[[str, bool], Any]
     match index_type:
         case "decrement":
             func = dec_index
         case "single_stock_total_return":
             func = ss_index
-        case "multi_basket_no_corax":
-            func = mult_index
         case _:
```

Comment:
i made this change because choice prohibits this from happening so this code was always unreachable and therefore uncoverable in tests